modules/upgrade-upgrading-cli.adoc

```diff
@@ -0,0 +1,147 @@
+// Module included in the following assemblies:
+//
+// * updating/updating-cluster.adoc
+
+[id="upgrade-upgrading-cli_{context}"]
+= Updating a cluster by using the CLI
+
+If updates are available, you can update your cluster by using the
+OpenShift CLI (`oc`).
+
+You can find information about available {product-title} advisories and updates
+link:https://access.redhat.com/downloads/content/290/ver=3.11/rhel---7/3.11.98/x86_64/product-errata[in the errata section]
+of the Customer Portal.
+
+[IMPORTANT]
+====
+If you installed the OpenShift 4 Beta Drop 5, you can update to {product-title}
+{product-version}.
+====
+
+.Prerequisites
+
+* Install the OpenShift Command-line Interface (CLI), commonly known as `oc`.
+* Log in to the cluster as user with `cluster-admin` privileges.
+* Install the `jq` package.
+
+.Procedure
+
+. Ensure that your cluster is available:
++
+----
+$ oc get clusterversion
+
+NAME      VERSION     AVAILABLE   PROGRESSING   SINCE     STATUS
+version   4.1.0-rc.4   True        False         24h       Cluster version is 4.1.0-rc.4
+----
++
+If your `VERSION` is `4.1.0-rc.4`, you installed OpenShift 4 Beta Drop 5, and
+you can update to {product-title} {product-version}.
+
+. Review the current update channel information and confirm that your channel
+is set to `stable-4.1`:
++
+----
+$ oc get clusterversion -o json|jq ".items[0].spec"
+
+{
+  "channel": "stable-4.1",
+  "clusterID": "53f97452-9856-4a3c-8260-00c1de2668a1",
+  "upstream": "https://api.openshift.com/api/upgrades_info/v1/graph"
+}
+----
+
+. View the available updates and note the version number of the update that
+you want to apply:
++
+----
+$ oc adm upgrade
+
+Cluster version is 4.0.0-0.nightly-2019-01-25-200832
+Updates:
```

Comment:
The payload version in your example may confuse users due to there are mixed prerelease(nightly) and release(rc) version. And they can not be mixed according to current update process. Could u update them to with either prerelease or release payload.

Reply:
@wking, do you have suggestions about getting better command output for upgrades before GA? I never see anything helpful in the stable channel.

Reply:
`stable-4.0` has some edges if you want to launch a [4.0.0-0.9 cluster][1] to see them.  Currently no 4.1 edges:

```console
$ curl -sH 'Accept:application/json' https://api.openshift.com/api/upgrades_info/v1/graph?channel=stable-4.1 | jq .
{
  "nodes": [],
  "edges": []
}
$ curl -sH 'Accept:application/json' https://api.openshift.com/api/upgrades_info/v1/graph?channel=prerelease-4.1 | jq .edges
[]
```

We're supposed to have a 4.1 upgrade at some point leading into GA, but we keep making breaking changes without operator migration support.  Maybe there will be an rc.5->rc.6 edge in `prerelease-4.1` that you can use in the next few days.

[1]: https://github.com/openshift/installer/releases/tag/v0.16.1